modules/sql/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "name" {
```

Comment:
Aici probabil in loc de nume o sa avem nevoie de output un connection string pentru jdbc cu database-ul respectiv, sau connection string plus database name separat, trebuie vazut.

Reply:
Am dat output la un string care include si baza de date. Daca este nevoie pot da output si separat la numele bazei de date. (m-am luat dupa tutorialul [asta](https://www.mysqltutorial.org/mysql-jdbc-tutorial/connecting-to-mysql-using-jdbc-driver/) cand am unit stringul)